WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingTagsViewModel.kt

```diff
@@ -38,8 +39,9 @@ class PrepublishingTagsViewModel @Inject constructor(
     private val _toolbarTitleUiState = MutableLiveData<UiString>()
     val toolbarTitleUiState: LiveData<UiString> = _toolbarTitleUiState
 
-    fun start(editPostRepository: EditPostRepository) {
+    fun start(editPostRepository: EditPostRepository, dontCloseKeyboard: Boolean = false) {
```

Comment:
Hey @mzorz I was wondering if it would be better to call this `closeKeyboard` and then flip the value before it's used in `onBackButtonClicked()`? Let me know

Reply:
nice, changed the name and meaning in c24f425087